.github/workflows/build.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         - uses: actions/checkout@v2
         - uses: actions/setup-node@v2
           with:
-            node-version: 18
+            node-version: 16
         - run: npm install -g @adobe/aio-cli
         - run: npm install
         - run: npm run lint
```

Comment:
so we always want the build/test to run only on stage? Referring to line 25

Reply:
I think that is just boilerplate from the AIO-CLI team tests.  I would have to go back and look at its history.